### PR TITLE
Add  method for Config

### PR DESCRIPTION
**Pull request by stefanv**
@Theodlz This should make your life a bit simpler in your default roles PR.

Now you can do `cfg.get('user.default_roles', [])`

### CODE REVIEW

**Comment by Theodlz** [comment]
Very nice ! Thanks a lot.

File: app/config.py [hunk 1]
```diff
@@ -63,12 +63,19 @@ def __getitem__(self, key):
...
-            if val is None:
-                return None
+            if isinstance(val, dict):
+                val = dict.__getitem__(val, key)
+            else:
+                raise KeyError(key)
 
         return val
 
+    def get(self, key, default=None, /):
```
Comment (profjsb) [inline app/config.py:73]: I don't understand the need for a slash at the end here.

Comment (stefanv) [reply app/config.py:73]: It is the same signature as what Python has for its dictionary; it means you cannot specify any arguments before the slash as keyword arguments.  I.e., you cannot write `get(key, default='foo')` but have to write `get(key, 'foo')`.  I'm just following the existing Python pattern here (personally, I don't really see the point).

Comment (profjsb) [reply app/config.py:73]: Thanks for that explanation

**Review by profjsb — APPROVED** [review]

**Pull request merged**